### DISCUSSION TITLE
Ability to have a Public Repo, with Private files

### DISCUSSION THREAD

**Discussion by iced-queen** [community/community#80858] [Repositories]
### Select Topic Area

Product Feedback

### Body

So currently you can make either a Public or Private repo. (I'm new to GitHub, so I may be wrong).
And I thought it'd be nice and at least to me, useful, if you were able to make a Public repo, but have specific files Private in there. So anyone can view the Repo, but specified files can only be viewed by Repo owners / contrubutors.

**Comment by Ayesha38600** [comment]
GitHub does not support making individual files within a public repository private. Repositories on GitHub are either public (accessible to everyone) or private (accessible to collaborators with permission).

**Reply by iced-queen** [reply]
That's the point with my suggestion. 😅
I'd love to see the option to make individual files private within a public repository
